src/tests/unit/tests/scanner/axe-utils.test.ts

```diff
@@ -324,132 +323,78 @@ describe('AxeUtils', () => {
 
     describe('hasBackgoundImage', () => {
         let fixture: HTMLElement;
-        let windowMock: IGlobalMock<typeof window.getComputedStyle>;
 
         beforeEach(() => {
             fixture = createTestFixture('test-fixture', '');
-            windowMock = GlobalMock.ofInstance(
-                window.getComputedStyle,
-                'getComputedStyle',
-                window,
-                MockBehavior.Strict,
-            );
         });
 
         afterEach(() => {
             document.body.querySelector('#test-fixture').remove();
         });
 
-        it('has bg image', () => {
+        it('returns true with background: some-value', () => {
             fixture.innerHTML = `
-                <img id="el1" alt="" />
+                <img id="el1" alt="" style="background: \\"some-value\\"" />
```

Comment:
I like that by removing the windowMock, it reduces the abstraction and improves the readability here.

Reply:
I agree, I think most of the tests ended up more readable overall.